src/js/App/RootApp/RootApp.js

```diff
@@ -12,11 +12,73 @@ import useQuickstartsStates from '../QuickStart/useQuickstartsStates';
 import { populateQuickstartsCatalog } from '../../redux/actions';
 import { LazyQuickStartCatalog } from '../QuickStart/LazyQuickStartCatalog';
 import useHelpTopicState from '../QuickStart/useHelpTopicState';
+import Notifications from '../Notifications';
+
+const useEvents = () => {
+  console.log('Use events was for some reason triggered');
+  const events = useRef({});
+  const conn = useRef();
+  const activeModule = useSelector(({ chrome: { activeModule } }) => activeModule);
+  const dispatch = useDispatch();
+  function handleEvent(payload) {
+    if (payload.type === 'new-notification') {
+      dispatch({ type: 'add-notification', payload });
+    } else if (payload.type === 'invalidate') {
+      Object.values(events.current)
+        .flat()
+        .forEach((cb) => {
+          cb(payload);
```

Comment:
I'd check if the callback is defined in here
```suggestion
          cb?.(payload);
```

Reply:
It's checked when a callback is registered. It will not register listener if a callback is not a function